contributors/design-proposals/api-machinery/admission-webhook-GA-proposal.md

```diff
@@ -0,0 +1,488 @@
+# Graduate Admission Webhooks to GA
+
+Authors: @mbohlool
+
+Reviewers: @liggit @deads2k @sttts @caesarxuchao
+
+Sig: api-machinery
+
+## Summary
+
+Admission webhooks are a way to extend kubernetes by putting hooks over object creation/modification. Admission webhooks can mutate or validate the object. This feature has been Beta since Kubernetes 1.9. This document outline required steps to graduate it to GA.
```

Comment:
Part of the KEP template will be graduation criteria, which also include things like testing and documentation.

Including the open questions we still need to resolve in this list would help readers of the document understand the current state, and allow quicker iteration/approval to get agree-on aspects approved for work in 1.14 while still reflecting accurately the decisions that still need to be made.

I like the "this is what the v1 API would look like" section (it's useful to get the big picture), but as long as there are potentially API-affecting open questions, it should note that the described v1 API is pending resolution of those questions.